### DISCUSSION TITLE
detectron2 on anime images

### DISCUSSION THREAD

**Discussion by frozen-mind** [facebookresearch/detectron2#4529] [Ideas]
have anyone ever tried to use detectron2 on anime images for instance segmentation?
i tried to train the model on custom dataset, and the result is some how weird...
for a same image, the segmentation result is unstable, like this:
![8b754ba064a47f7684fff878a6e32e2](https://user-images.githubusercontent.com/56190871/188867211-60250875-3f6a-4a07-83cc-39a9dec539fd.jpg)
![image](https://user-images.githubusercontent.com/56190871/188867326-7b6fbbe2-1fe5-492a-af34-2c710e168a6b.png)
is it because of lack of training data? or lack of training iterations?

**Comment by TomLucidor** [comment]
Use a dedicated model for anime please, as Facebook mainly targets real life task. For example:
- https://github.com/SkyTNT/anime-segmentation
- https://github.com/zymk9/Yet-Another-Anime-Segmenter
- https://github.com/CartoonSegmentation/CartoonSegmentation
